samples/basic/threads/README.rst

```diff
@@ -8,11 +8,15 @@ Overview
 
 This example demonstrates spawning of multiple threads using K_THREAD_DEFINE.
 
-The example works by spawning  three threads. The first two threads control a
+The example works by spawning three threads. The first two threads control a
 separate LED. Both of these LEDs (USR1 and USR2) have their individual loop
 control and timing logic defined by separate functions.
 
-The third thread, uart_out(), sends out messages on the UART using printk().
+After a thread has toggled an LED it will push information about what
+thread toggled the LED and how many times it has done into a FIFO.
+
+The third thread, uart_out(), will printk the information that comes
```

Comment:
How about:

    The third thread, ``uart_out()``, uses printk (over the UART) to display
    the information that comes through the FIFO.

Reply:
Done.